dotnet/test/common/Environment/EnvironmentManager.cs

```diff
@@ -60,8 +61,13 @@ private EnvironmentManager()
 
             string activeDriverConfig = System.Environment.GetEnvironmentVariable("ACTIVE_DRIVER_CONFIG") ?? TestContext.Parameters.Get("ActiveDriverConfig", env.ActiveDriverConfig);
             string driverServiceLocation = System.Environment.GetEnvironmentVariable("DRIVER_SERVICE_LOCATION") ?? TestContext.Parameters.Get("DriverServiceLocation", env.DriverServiceLocation);
-
             string browserLocation = System.Environment.GetEnvironmentVariable("BROWSER_LOCATION") ?? TestContext.Parameters.Get("BrowserLocation", string.Empty);
+            string enableDebugging = System.Environment.GetEnvironmentVariable("DEBUG") ?? TestContext.Parameters.Get("Debug", env.Debug);
+
+            if (!string.IsNullOrEmpty(enableDebugging))
+            {
+                Log.SetLevel(LogEventLevel.Debug);
```

Comment:
It is already set to `Trace` level here by default: https://github.com/SeleniumHQ/selenium/blob/1cfa0a0493ff2ad83fbf37bd5a55c9501d46c333/dotnet/test/common/AssemblyFixture.cs#L37

Usually I see RBE logs directly in EngFlow:

![image](https://github.com/user-attachments/assets/59c024bc-2b17-405d-bbe2-30d95cddc6e8)

Reply:
Ah, ok. Why isn't it showing more output when the tests are passing?

In general the behavior I'm proposing is for the logging not to be turned on by default for faster execution / cleaner logs, and then allow the user to turn that on when needed (or automatically when failing). But it looks like current .NET is already not logging unnecessarily, so curious how you have it working.